Traceback (most recent call last):
  File "/usr/share/miniconda/envs/DevelopmentalRegulation/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/share/miniconda/envs/DevelopmentalRegulation/lib/python3.8/site-packages/nbclient/client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/share/miniconda/envs/DevelopmentalRegulation/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/share/miniconda/envs/DevelopmentalRegulation/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/usr/share/miniconda/envs/DevelopmentalRegulation/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/share/miniconda/envs/DevelopmentalRegulation/lib/python3.8/site-packages/nbclient/client.py", line 535, in async_execute
    await self.async_execute_cell(
  File "/usr/share/miniconda/envs/DevelopmentalRegulation/lib/python3.8/site-packages/nbclient/client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/share/miniconda/envs/DevelopmentalRegulation/lib/python3.8/site-packages/nbclient/client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# print(sil_score)
print(max(ys))
print(ys.count(max(ys)))
print(xs[ys.index(max(ys))])
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-17-c3d4fbb1e4e1>[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;31m# print(sil_score)[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mprint[0m[0;34m([0m[0mmax[0m[0;34m([0m[0mys[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      3[0m [0mprint[0m[0;34m([0m[0mys[0m[0;34m.[0m[0mcount[0m[0;34m([0m[0mmax[0m[0;34m([0m[0mys[0m[0;34m)[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0mprint[0m[0;34m([0m[0mxs[0m[0;34m[[0m[0mys[0m[0;34m.[0m[0mindex[0m[0;34m([0m[0mmax[0m[0;34m([0m[0mys[0m[0;34m)[0m[0;34m)[0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'ys' is not defined
NameError: name 'ys' is not defined

